apps/src/geographyConstants.js

```diff
@@ -54,250 +54,249 @@ export const STATES = [
-  "French Guiana",
-  "Guernsey",
-  "Ghana",
-  "Gibraltar",
-  "Greenland",
-  "Gambia",
-  "Guinea",
-  "Guadeloupe",
-  "Equatorial Guinea",
-  "Greece",
-  "South Georgia and the South Sandwich Islands",
-  "Guatemala",
-  "Guam",
-  "Guinea-Bissau",
-  "Guyana",
-  "Hong Kong",
-  "Heard Island and McDonald Islands",
-  "Honduras",
-  "Croatia",
-  "Haiti",
-  "Hungary",
-  "Indonesia",
-  "Ireland",
-  "Israel",
-  "Isle of Man",
-  "India",
-  "British Indian Ocean Territory",
-  "Iraq",
-  "Islamic Republic of Iran",
-  "Iceland",
-  "Italy",
-  "Jersey",
-  "Jamaica",
-  "Jordan",
-  "Japan",
-  "Kenya",
-  "Kyrgyzstan",
-  "Cambodia",
-  "Kiribati",
-  "Comoros",
-  "Saint Kitts and Nevis",
-  "Democratic People's Republic of Korea",
-  "Republic of Korea",
-  "Kuwait",
-  "Cayman Islands",
-  "Kazakhstan",
-  "Lao People's Democratic Republic",
-  "Lebanon",
-  "Saint Lucia",
-  "Liechtenstein",
-  "Sri Lanka",
-  "Liberia",
-  "Lesotho",
-  "Lithuania",
-  "Luxembourg",
-  "Latvia",
-  "Libya",
-  "Morocco",
-  "Monaco",
-  "Moldova",
-  "Montenegro",
-  "Madagascar",
-  "Marshall Islands",
-  "Republic of Macedonia",
-  "Mali",
-  "Myanmar",
-  "Mongolia",
-  "Macao",
-  "Northern Mariana Islands",
-  "Martinique",
-  "Mauritania",
-  "Montserrat",
-  "Malta",
-  "Mauritius",
-  "Maldives",
-  "Malawi",
-  "Mexico",
-  "Malaysia",
-  "Mozambique",
-  "Namibia",
-  "New Caledonia",
-  "Niger",
-  "Norfolk Island",
-  "Nigeria",
-  "Nicaragua",
-  "Netherlands",
-  "Norway",
-  "Nepal",
-  "Nauru",
-  "Niue",
-  "New Zealand",
-  "Oman",
-  "Panama",
-  "Peru",
-  "French Polynesia",
-  "Papua New Guinea",
-  "Philippines",
-  "Pakistan",
-  "Poland",
-  "Saint Pierre and Miquelon",
-  "Pitcairn",
-  "Puerto Rico",
-  "Palestine",
-  "Portugal",
-  "Palau",
-  "Paraguay",
-  "Qatar",
-  "Réunion",
-  "Romania",
-  "Serbia",
-  "Russian Federation",
-  "Rwanda",
-  "Saudi Arabia",
-  "Solomon Islands",
-  "Seychelles",
-  "Sudan",
-  "Sweden",
-  "Singapore",
-  "Slovenia",
-  "Svalbard and Jan Mayen",
-  "Slovakia",
-  "Sierra Leone",
-  "San Marino",
-  "Senegal",
-  "Somalia",
-  "Suriname",
-  "South Sudan",
-  "Sao Tome and Principe",
-  "El Salvador",
-  "Sint Maarten (Dutch part)",
-  "Syrian Arab Republic",
-  "Swaziland",
-  "Turks and Caicos Islands",
-  "Chad",
-  "Togo",
-  "Thailand",
-  "Tajikistan",
-  "Tokelau",
-  "Timor-Leste",
-  "Turkmenistan",
-  "Tunisia",
-  "Tonga",
-  "Turkey",
-  "Trinidad and Tobago",
-  "Tuvalu",
-  "Taiwan",
-  "Tanzania",
-  "Ukraine",
-  "Uganda",
-  "United States",
-  "Uruguay",
-  "Uzbekistan",
-  "Holy See (Vatican City State)",
-  "Saint Vincent and the Grenadines",
-  "Bolivarian Republic of Venezuela",
-  "Virgin Islands, British",
-  "Virgin Islands, U.S.",
-  "Viet Nam",
-  "Vanuatu",
-  "Wallis and Futuna",
-  "Samoa",
-  "Yemen",
-  "Mayotte",
-  "South Africa",
-  "Zambia",
-  "Zimbabwe",
+  {label: "AD", value: "Andorra"},
+  {label: "AE", value: "United Arab Emirates"},
```

Comment:
Presumably nothing else is using this, so it's safe to change? More importantly...

Why do we need this `COUNTRIES` array? Why not reuse the `COUNTRY_CODE_TO_COUNTRY_NAME ` hash and helper methods in https://github.com/code-dot-org/code-dot-org/blob/staging/lib/country_codes.rb? 

Note, you can generate the `{label: , value:}` array from the other country hash with something like
```ruby
COUNTRY_CODE_TO_COUNTRY_NAME.map{|code, name| {label: code, value: name}}
```

Reply:
How do I access that hash and helper methods from my component?

Reply:
Since it's under the project root lib (outside of `dashboard`) it's not autoloaded, but the path is in scope. You just need to require the file, `require 'country_code'`. Here's an example: https://github.com/code-dot-org/code-dot-org/blob/61466573061da2554f959c0b0837982bb587249a/dashboard/app/views/shared/_school_info_interstitial.html.haml#L1